source/includes/streaming-distinction.rst

```diff
@@ -0,0 +1,3 @@
+.. important::
+
+   `Spark Structured Streaming <https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html>`__ and `Spark Streaming with DStreams <https://spark.apache.org/docs/latest/streaming-programming-guide.html>`__ are different.
```

Comment:
On second thought, I think this is useful as a simple, reusable note to get users to confirm they're reading about what they want to be reading about. Discussing the differences in this note would expand the scope of the page a little bit and require mentioning RDDs, which I've tried to avoid in v10.

Reply:
SGTM!